ocean-components/src/main/java/br/com/useblu/oceands/components/compose/OceanAlert.kt

```diff
@@ -378,12 +384,30 @@ fun OceanAlertEntitledShort(
         Column(
             modifier = Modifier.weight(1f)
         ) {
-            OceanText(
-                text = title,
-                style = style.titleStyle.invoke(),
-                color = style.titleColor.invoke(),
-                maxLines = 2
-            )
+            Row (
+                verticalAlignment = CenterVertically
+            ){
+                OceanText(
+                    text = title,
+                    style = style.titleStyle.invoke(),
+                    color = style.titleColor.invoke(),
+                    maxLines = 2
+                )
+
+                OceanSpacing.StackXXXS()
+
+                if (!tooltip.isNullOrBlank()) {
```

Comment:
Daria pra ser `if (tooltip == null)` ?

Reply:
é negação, só mostra se não for null ou empty